lib/postprocessing/image.js

```diff
@@ -39,17 +39,28 @@ const SUPPORTED_OUTPUT_FMT = {
 // Note: order is important and defines priority for intermediate format - first entry will be most preferred
 const SUPPORTED_INPUT_TYPES = {
     "TIFF": "tif",  // most preferred format as it allows to preserve many features
-    "PNG":  "png",  // lossless over
+    "PNG": "png",  // lossless over
     "JPEG": "jpg",  // ..lossy
-    "GIF":  "gif",  // (should never get here in practice)
-    "BMP":  "bmp"
+    "GIF": "gif",  // (should never get here in practice)
+    "BMP": "bmp",
+    "SVG": "svg"
 };
 const SUPPORTED_INPUT_FORMAT_IMAGEMAGICK = Object.keys(SUPPORTED_INPUT_TYPES);
 const SUPPORTED_INPUT_FMT = Object.values(SUPPORTED_INPUT_TYPES);
 const DEFAULT_WORKER_SUPPORTED_OUTPUT_FMT = [
     "png"
 ];
 
+const FORMAT_SUPPORTS_TRANSPARENCY = {
+    "png": true,
+    "tif": true,
+    "tiff": true,
+    "gif": true,
+    "bmp": true,
+    "svg": true
+};
+const DEFAULT_BACKGROUND_COLOR = "#FFFFFF";
```

Comment:
Probably should change this from `#FFFFFF` to `rgba(0, 0, 0, 0.0)`

Reply:
`rgba(0, 0, 0, 0.0)` failed during testing for me. Any special way to add quotes around it that I may be missing?

Reply:
Nevermind! It's changed.